net.sourceforge.roccat.roccat-tools.yml

```diff
@@ -0,0 +1,66 @@
+app-id: net.sourceforge.roccat.roccat-tools
+runtime: org.gnome.Platform
```

Comment:
does there really need the GNOME runtime since it's building gtk2?

Reply:
You are right, I switched to `org.freedesktop.Platform`, adding `libnotify` manually and `gudev` from `shared-modules`.